scripts/fonts.json

```diff
@@ -50,7 +50,10 @@
       [126208, 126287],
       [126464, 126719]
     ],
-    "armenian": [[1328, 1423]],
+    "armenian": [
+      [1328, 1423],
+      [64272, 64279]
```

Comment:
Add part of [Alphabetic Presentation Forms](https://en.wikipedia.org/wiki/Alphabetic_Presentation_Forms), moved from `latin`